modules/statik/src/variables/StatikVariable.php

```diff
@@ -51,4 +53,14 @@ public function slugify(string $string): string
     {
         return ElementHelper::generateSlug($string);
     }
+
+    public function hyphenate($string, $minimumWordLength = 12)
```

Comment:
@KarelJanVanHaute Kan je deze functie correct typehinten? We proberen dat nu overal consistent te doen.

Reply:
Deze ganse functie kan misschien in een cache tag komen, om te vermijden dat we dat voor dezelfde tekst steeds opnieuw gaan uitvoeren.

Reply:
Deze heb ik toegevoegd.